.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
You typically don't really check in files like this (IDE settings) but it's fine in this case. If you ever want git to ignore stuff, look into creating a `.gitignore` file.